src/apps/recommended-material/RecommendedMaterial.dev.tsx

```diff
@@ -17,6 +18,10 @@ export default {
       defaultValue: "work-of:870970-basis:136336282",
       control: { type: "text" }
     },
+    materialType: {
+      defaultValue: "bog",
+      control: { type: "select", options: DisplayMaterialTypeOptions }
+    },
```

Comment:
Consider adding a story without a preselected material type to showcase the consequences of using this.

Using a work where we know that there will be a significant difference in the presentation between material types would also be good. This might already be the case. If so: Carry on :smile:.

Reply:
I found one with 3 differences 🚀 

Also added a story with an "undefined" materialType. 👀